packages/schema-extract/src/extract-schema.ts

```diff
@@ -4,18 +4,31 @@ import typescript from 'typescript';
 import {LocalFileSystem} from 'kissfs';
 import {transform} from './file-transformer';
 import {createHost} from './isomorphc-typescript-host';
+import {SchemaLinker} from './file-linker';
 
-export function* extractSchema(basePath: string, filesGlob: string) {
+export function* extractSchema(basePath: string, filesGlob: string, linked = false) {
```

Comment:
the linked schema should be moved to a different function.

it requires the name of the export to link

Reply:
I created a new function and changed it back. Anyway the cli command doesn't work properly yet and I will fix it separately